src/test/regress/expected/multi_with_prepare.out

```diff
@@ -0,0 +1,339 @@
+-- prepared statements
+PREPARE prepared_test_1 AS
```

Comment:
can you add some parameters to the prepared statements?

Reply:
Ah, it was planned though, but then it was not supported yesterday yet :D Adding them too